dspace-xmlui-mirage2/src/main/webapp/styles/shared/_dspace-bootstrap-tweaks.scss

```diff
@@ -139,4 +139,9 @@ span.bold {
 /* Preserve whitespace. Used to display error stack in a readable way. */
 p.pre {
     white-space:pre;
+}
+.break-all{
+  -ms-word-break:   break-all;
+  word-break:       break-word;
```

Comment:
While testing #1420 I found that word-break must be set to break-all. (http://www.w3schools.com/cssref/css3_pr_word-break.asp)